frontend/app/src/premium/register-components.ts

```diff
@@ -126,6 +127,8 @@ export function registerComponents(app: App): void {
   // Version 26 - 1.34
   app.component('RefreshButton', RefreshButton);
   app.component('AssetBalanceStatisticSourceSetting', AssetBalanceStatisticSourceSetting);
+
+  app.component('MissingDailyPrices', MissingDailyPrices);
```

Comment:
We need to find the version number constant bump it and also add a note here since this is a breaking change. to keep track on which version this was introduced as we do before

No way around this?

Reply:
You mean if we can remove this at all?
Hmm it's pretty complicated to move to the premium-components itself :5754pepehmm: